Microsoft.Toolkit.Uwp.UI/Extensions/FrameworkElement/FrameworkElementExtensions.RelativeAncestor.cs

```diff
@@ -0,0 +1,94 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Uwp.UI.Extensions
+{
+    /// <summary>
+    /// Provides attached dependency properties for the <see cref="Windows.UI.Xaml.FrameworkElement"/>
+    /// </summary>
+    public static partial class FrameworkElementExtensions
```

Comment:
This is a lot of work to go through just to set/get the TabView for a TabViewItem. Just set an internal property within `GetContainerForItemOverride`

Reply:
This isn't used in the TabViewItem but in the other header templates.  This is also a stop-gap since Binding doesn't support RelativeAncestor, so it has other uses outside of just this control.